src/main/java/frc/robot/commands/ArcadeDrive.java

```diff
@@ -0,0 +1,83 @@
+// RobotBuilder Version: 3.1
+//
+// This file was generated by RobotBuilder. It contains sections of
+// code that are automatically generated and assigned by robotbuilder.
+// These sections will be updated in the future when you export to
+// Java from RobotBuilder. Do not put any code or make any change in
+// the blocks indicating autogenerated code or it will be lost on an
+// update. Deleting the comments indicating the section will prevent
+// it from being updated in the future.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+import frc.robot.subsystems.Drivetrain;
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=IMPORTS
+
+/**
+ *
+ */
+public class ArcadeDrive extends CommandBase {
+
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_DECLARATIONS
+    private final Drivetrain m_drivetrain;
+    private final XboxController m_xboxcontroller;
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_DECLARATIONS
+
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
+
+
+    public ArcadeDrive(Drivetrain subsystem, XboxController xboxcontroller) 
+    {
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
+        // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_SETTING
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=VARIABLE_SETTING
+        // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+        m_drivetrain = subsystem;
+        addRequirements(m_drivetrain);
+
+        m_xboxcontroller = xboxcontroller;
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+    }
+
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+    }
+
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+        //Get xbox controller info and pass to ArcadeDrive here!
+        //pass in trigger for y if forza drive wants to be implemented
+        m_drivetrain.arcadeDrive(m_xboxcontroller.getY(Hand.kLeft), m_xboxcontroller.getX(Hand.kLeft)); 
```

Comment:
This is good. This is where would would bring in the use of the triggers or the right thumbstick for control